File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/11/11.py
# Q11: Print numbers in increasing sequence (1, 12, 123, 1234, 12345).
# Input: An integer n
# Output: Number sequence pattern

n = int(input())
for i in range(1, n + 1):
    for j in range(1, i + 1):
        print(j, end="")
    print()

File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/11/11.go
// Q11: Print numbers in increasing sequence (1, 12, 123, 1234, 12345).
// Input: An integer n
// Output: Number sequence pattern

package main

import (
    "fmt"
    "strings"
)

func main() {
    var n int
    fmt.Scan(&n)
    for i := 1; i <= n; i++ {
        row := ""
        for j := 1; j <= i; j++ {
            row += strings.Repeat("", 0) + fmt.Sprintf("%d", j)
        }
        fmt.Println(row)
    }
}
